### DISCUSSION TITLE
2022 SSU Engineering Days 이벤트를 함께만들어갈 분을 구합니다.

### DISCUSSION THREAD

**Discussion by acious** [Soongsil-Developers/.github#12] [구인, 구직, 채용 공고]
마침 이 커뮤니티가 생겨서 더 다양한분들과 함께 숭머의 엔지니어링 컬쳐를 되살리기 좋다고 생각해서 이 이벤트 주최 계획을 공개드리고
함께하고싶은 재학생, 졸업생분들을 모집하려합니다.

https://docs.google.com/document/d/1yAtNNMx4LQzlRfmPrWXASVnIIKyAuCxBPhwRMYfwXHY/edit#heading=h.295ol5v0tepn

자세한 내용은 위 G Docs 문서를 봐주시면 좋을거같구요.
이 이벤트는 철저히 재학생들의 엔지니어링 경험과, 커리어 문제를 졸업생들이 함께 고민하고 해결해주고 이러한 커뮤니티의 형성을 목표로 진행하려던 이벤트였습니다.

이러한 목적을 이루는 이벤트를 올해 여름을 타겟으로 준비해보고 진행해볼 사람들을 구합니다.
이 글의 댓글도 좋고, ace.jckim@gmail.com 으로의 연락도 좋습니다!

**Comment by ghost** [comment]
정말 좋아보이는데 졸업생이 참가자로 나서도 되나요?

**Reply by acious** [reply]
경험을 어느정도 있어야해서 주최는 졸업생이 꼭 필요합니다!

**Reply by ghost** [reply]
많이 부족한 실력이지만 졸업생으로 참여할 수 있으면 참여하겠습니다 ㅎㅎ

**Reply by acious** [reply]
다음 미팅이 진행될때 연락드릴게요.
Github에 보이는 메일로 연락드리면 될까요~?

**Reply by ghost** [reply]
dongho5309@naver.com으로 연락주시면 감사드리겠습니다. @acious

**Comment by aube-dev** [comment]
저도 돕고 싶어요! 재학생이기도 하고 제 실력으로 어느 역할이 될지는 모르겠지만 웹프론트와 디자인 쪽에 나서볼게요

**Reply by acious** [reply]
넵 저희끼리 이야기할수있는 inner channel 통해서 연락드릴게요~
